phoenicis-scripts/src/main/java/org/phoenicis/scripts/wizard/UiSetupWizardImplementation.java

```diff
@@ -283,8 +283,9 @@ public String browse(String textToShow) {
      */
     @Override
     public String browse(String textToShow, String directory, List<String> allowedExtensions) {
+        final List<String> copiedAllowedExtensions = allowedExtensions != null ? List.copyOf(allowedExtensions) : null;
```

Comment:
Are you sure it should be null and not an empty List?

Reply:
It's the same behavior like before.